assets/tests/components/map.test.tsx

```diff
@@ -293,6 +298,90 @@ describe("<MapFollowingPrimaryVehicles />", () => {
     expect(onClick).not.toHaveBeenCalled()
   })
 
+  test("starts out with no dropdown menu", async () => {
+    jest.mocked(getTestGroups).mockReturnValue([TestGroups.DetoursPilot])
```

Comment:
These tests assert on the wrong use case. Gotta move them

Reply:
So moved: https://github.com/mbta/skate/pull/2373/commits/f3f5131d808a44465915323514641d675aa4809a